### ISSUE TITLE
Android emulator command ignores avd config.ini settings

### ISSUE THREAD

**Issue by kfieldi5** [travis-ci/travis-ci#9537]
I need to spin up an Android emulator with > 512mb of RAM. I had tried setting the `-memory 2048` flag but to no avail. I also noticed I will further need to tweak screen density, size and heapSize to be able to have our Espresso screenshot tests run smoothly. So I created a dump to the /Users/travis/.android/avd/test.avd/config.ini that overwrites the necessary values, leaving the ini file looking like 

```
d.ini.encoding=UTF-8
abi.type=armeabi-v7a
hw.cpu.arch=arm
hw.cpu.model=cortex-a8
hw.lcd.height=1280
hw.lcd.width=800
hw.lcd.density=320
hw.ramSize=2048
image.sysdir.1=system-images/android-21/default/armeabi-v7a/
skin.name=WVGA800
skin.path=platforms/android-21/skins/WVGA800
tag.display=Default
tag.id=default
vm.heapSize=128
```

Unfortunately travis does not respect this config.ini file when spinning up said emulator. Not sure how/why, all tests I've done locally and on other build servers shows that this config file being read is simply part of the ./emulator command.

Long story short, even when acting within travis limitations (no skin, no gpu, etc.) there is no way to customize the emulator.

Please help!

**Comment by kfieldi5** [comment]
Seems to be due to a limitation of the goldfish kernel. Rebuilding the kernel a bit too expensive of operation for our CI. Newer API's (>= 24) seem to respect the memory specified via config.ini
